pontoon/checks/tasks.py

```diff
@@ -0,0 +1,85 @@
+import logging
+import time
+
+from celery import shared_task
+
+from django.db import transaction
+
+from pontoon.checks import libraries
+from pontoon.checks.utils import get_failed_checks_db_objects
+from pontoon.checks.models import (
+    Warning,
+    Error,
+)
+
+from pontoon.base.models import Translation
```

Comment:
Nit: We should keep the imports. in the alphabetic order, so this one should go above the `from pontoon.checks import libraries`. Sorry I missed that earlier.